registry/tests/templates.spec.ts

```diff
@@ -576,6 +576,57 @@ describe(`Tests ${example.url}`, () => {
         });
     });
 
+    describe('Partial Update', () => {
+        it("should not partial update any record if record doesn't exist", async () => {
+            const incorrect = { name: 123 };
```

Comment:
`incorrect` -> `nonExisting`, slightly better semantics on why it's incorrect, some might assume that's due to int inside name